docs/.vuepress/config/locales/it/nav.js

```diff
@@ -0,0 +1,13 @@
+module.exports = [
+    { text: 'Guide', link: '/it/' },
+    { text: 'Extend', link: '/extend/' },
```

Comment:
Since extend docs are now translated, this should be switched back to '/it/extend/'